Parser/convertSBEcnvVar.m

```diff
@@ -122,12 +122,23 @@
     case 'flC'
       name = 'CPHL';
       comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements. Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
-      
+
+    %fluorescence (ug/l)
+    %Aquatracka SBE19plus cdom sensor as chpl #550
+    case 'flCUVA'
+      name = 'CPHL';
+      comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements (Chelsea UV Aquatracka mounted on SBE19Plus). Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
```

Comment:
You don't need to include provenance from SBE19Plus. This is captured in global attribute + flCUVA code is going to be used whether the sensor is mounted on an SBE19 or SBE35.